Gemfile

```diff
@@ -2,13 +2,14 @@ source 'https://rubygems.org'
 
 ruby '2.2.3'
 
+gem 'faker'
 gem 'selenium-webdriver'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.6'
 # Use postgresql as the database for Active Record
 gem 'pg', '~> 0.15'
 # Use SCSS for stylesheets
-gem 'sass-rails', '~> 5.0'
+# gem 'sass-rails', '~> 5.0'
```

Comment:
Why are you getting rid of the explicit requirement of this gem? It's important, I think it's good to have it in your gemfile!